README.md

```diff
@@ -38,12 +38,15 @@ Here are the supported command-line options:
 
 ```
 USAGE:
-    docuum
+    docuum [OPTIONS]
```

Comment:
Did the output of `docuum --help` change here with the introduction of the new flag? (Is this `clap`'s behavior?)

Reply:
`clap` says[^1]:

> The default is that the auto-generated help message will group flags, and options separately.

Maybe the flag `--fast-fail` makes `clap` reorganize the help information.

[^1]: https://docs.rs/clap/2.34.0/clap/enum.AppSettings.html#variant.UnifiedHelpMessage